src/test/java/de/zalando/aruha/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -191,13 +194,41 @@ public void whenPUTRepoNakadiExceptionThen422() throws Exception {
         Mockito
                 .doThrow(NakadiException.class)
                 .when(repo)
-                .findByName("event-name");
+                .findByName(EVENT_TYPE_NAME);
 
-        putEventType(eventType, "event-name")
+        putEventType(eventType, EVENT_TYPE_NAME)
                 .andExpect(status().isUnprocessableEntity())
                 .andExpect(content().contentType("application/problem+json"))
                 .andExpect(content().string(matchesProblem(expectedProblem)));
     }
+    
+    @Test
+    public void canExposeASingleEventType() throws Exception {
+        final EventType expectedEventType = buildEventType();
+
+        when(repo.findByName(EVENT_TYPE_NAME)).thenReturn(expectedEventType);
+
+        final MockHttpServletRequestBuilder requestBuilder = get("/event-types/" + EVENT_TYPE_NAME)
+                .accept(APPLICATION_JSON);
+
+        mockMvc.perform(requestBuilder)
+                .andExpect(status().is2xxSuccessful())
```

Comment:
Why you have so broad check? (I also noticed that in some other of your tests)
I believe we can be more specific at checking status codes as we have specific status codes in our API.